isa-bel/react-native-workshop
# Feat - Add questions

#### What does this PR do?

Adds the Question and Message components and the navigation.

#### Where should the reviewer start?

In `src/components/Message/Message.js`, `src/components/Question/Question.js`, and `src/components/Home/Home.js` for the new components.

#### How should this be manually tested?
1. Update the dependencies.
    ```bash
    yarn
    ```
2. Run the test script.
    ```bash
    yarn test
    ```
3. Verify that there are no errors.
4. Load the iOS simulator:
    ```bash
    react-native run-ios
    ```
5. Verify that the navigation works as expected.

#### Any background context you want to provide?



#### What are the relevant tickets?



N/A

#### Checklist
